hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/rmadmin/TestableFederationRMAdminInterceptor.java

```diff
@@ -66,6 +67,11 @@ protected ResourceManagerAdministrationProtocol getAdminRMProxyForSubCluster(
         }
         mockRM.init(config);
         mockRM.start();
+        try {
+          mockRM.registerNode("127.0.0.1:1", 102400, 100);
+        } catch (Exception e) {
+          Assert.fail(e.getMessage());
```

Comment:
Do we get much by doing this? letting the exception surface would be ok.

Reply:
I will fix it.